repos/system_upgrade/el7toel8/actors/updategrubcore/libraries/library.py

```diff
@@ -0,0 +1,83 @@
+import os
+
+from leapp.libraries.stdlib import run, CalledProcessError
+from leapp.exceptions import StopActorExecutionError
+from leapp import reporting
+
+
+def has_grub(blk_dev):
+    """
+    Check whether GRUB is present on block device
+    """
+    try:
+        run(['dd', 'if={}'.format(blk_dev), 'of=/tmp/mbr', 'bs=1', 'count=512'])
```

Comment:
nitpick: you would want bs=512 & count=1 because you do 512 reads otherwise. Where as bs=512 gives a block size of 512

Reply:
aah, what have I done? Of course I wanted `bs=512 & count=1`. Not sure how my mind came up with this :)